stock_picking_auto_print/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Picking Auto Print",
+    "summary": "Print picking delivery slip automatically after validation",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "Odoo S.A., Tecnativa, Odoo Community Association (OCA)",
```

Reply:
I see that the code is different enough, and the core one doesn't include tests, that we can skip this attribution. Do you agree?

Reply:
Ok, it's true that although the API will be the same and no migration would work be needed in v17, the code adaptations had to be deep